scalyr_agent/scalyr_client.py

```diff
@@ -465,7 +469,7 @@ def close(self, current_time=None):
             self.__connection = None
             self.__last_connection_close = current_time
 
-    def add_events_request(self, session_info=None, max_size=1*1024*1024*1024):
+    def add_events_request(self, session_info=None, max_size=1*1024*1024*1024, disable_new_addevents_format=False):
```

Comment:
You already have a property to store `__disable_new_addevents_format` on the `ScalyrClientSession` object.  You shouldn't pass it in here.  In general, we should just specify that once when we create the client session object.